celesta-sql/src/main/java/ru/curs/celesta/score/NamedElement.java

```diff
@@ -83,7 +84,17 @@ public final int hashCode() {
 
     @Override
     public final boolean equals(Object obj) {
-        return obj instanceof NamedElement ? name.equals(((NamedElement) obj).getName()) : name.equals(obj);
+        if (obj == this) {
+            return true;
+        }
+        if (obj instanceof NamedElement) {
+            return Objects.equals(this.name, ((NamedElement) obj).getName());
+        }
+        if (obj instanceof String) {
```

Comment:
I think this clause can be  safely removed, if no tests fail.
This violates `equals` contract for no reason and SpotBugs is absolutely right nagging about it.

Reply:
Yes, let's try.